coral/src/app/features/approvals/components/ApprovalsLayout.tsx

```diff
@@ -27,17 +35,22 @@ function ApprovalsLayout(props: ApprovalsLayoutProps) {
           );
         })}
       </Box>
-
-      <Box
-        display={"flex"}
-        flexDirection={"column"}
-        alignItems={"center"}
-        rowGap={"l4"}
-        className={"a11y-enhancement-data-table"}
-      >
-        {table}
-        {pagination}
-      </Box>
+      {isLoading && <SkeletonTable />}
```

Comment:
Maybe this could be passed inside the body, so the loading of the table data does not block showing the filters? Also would be a more accurate skeleton state.

Like  on line 39 and next, have this:

```
      {errorMessage !== undefined && (
        <Alert type={"error"}>{errorMessage}. Please try again later!</Alert>
      )}
        <Box
          display={"flex"}
          flexDirection={"column"}
          alignItems={"center"}
          rowGap={"l4"}
          className={"a11y-enhancement-data-table"}
        >
       {isLoading 
           ? <SkeletonForm /> : 
           : <>{table}{pagination}</>
        }
        </Box>
```

Reply:
The filters are not blocked. It's only the element passed as `table` and the pagination (that depends on the same data) that is rendered conditionally.

Reply:
Ah yes, I'm sorry, I can't read ^^"